drivers/sensor/nrf5/Kconfig

```diff
@@ -8,3 +8,22 @@ config TEMP_NRF5
 	depends on SOC_FAMILY_NRF
 	help
 	  Enable driver for nRF5 temperature sensor.
+
+if TEMP_NRF5
+
+config TEMP_NRF5_SENSOR
```

Comment:
note that disabling `TEMP_NRF5_SENSOR` and `TEMP_NRF5_BETTER_ACCURACY` gives 688 bytes savings.